discretize/utils/mesh_utils.py

```diff
@@ -1051,6 +1051,106 @@ def active_from_xyz(mesh, xyz, grid_reference="CC", method="linear"):
     return active.ravel()
 
 
+def example_simplex_mesh(rect_shape):
```

Comment:
Nice. This makes sense.

Image that is _close_ if you make a publication soon .... :)
https://phd.row1.ca/phd/discretize#EzkybKpuA4